lib/datadog/appsec/waf.rb

```diff
@@ -182,6 +182,7 @@ class Object < ::FFI::Struct
         attach_function :ddwaf_object_get_unsigned, [:ddwaf_object], :uint64
         attach_function :ddwaf_object_get_signed, [:ddwaf_object], :int64
         attach_function :ddwaf_object_get_index, [:ddwaf_object, :size_t], :ddwaf_object
+        attach_function :ddwaf_object_get_bool, [:ddwaf_object], :bool
```

Comment:
I don't think we have spec coverage for these getters, it would be nice to have!

Reply:
Also there should be a corresponding update to `sig/datadog/appsec/waf.rbs`.

Reply:
I can add the spec coverage for those getters